packages/leancode_lint/lib/helpers.dart

```diff
@@ -29,6 +32,76 @@ Expression? maybeGetSingleReturnExpression(FunctionBody body) {
   };
 }
 
+bool isHook(Expression expression) {
```

Comment:
This doesnt check if the expression is a hook. It checks if there is a hook being used somewhere